masci_tools/io/cif2inp_ase.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+###############################################################################
+# Copyright (c), Forschungszentrum Jülich GmbH, IAS-1/PGI-1, Germany.         #
+#                All rights reserved.                                         #
+# This file is part of the Masci-tools package.                               #
+# (Material science tools)                                                    #
+#                                                                             #
+# The code is hosted on GitHub at https://github.com/judftteam/masci-tools    #
+# For further information on the license, see the LICENSE.txt file            #
+# For further information please visit http://www.flapw.de or                 #
+#                                                                             #
+###############################################################################
+
+"""
+Independent utility script to convert cif file formats to input for the inpgen code
+Usage: `python cif2inp_ase.py <filename.cif>`
+"""
+
+import sys
+import ase.io
+import numpy as np
+
+cifFilename = sys.argv[1]
+structure = ase.io.read(cifFilename)
+
+structureFormula = structure.get_chemical_formula()
+inpFilename = 'inp_'+structureFormula
+
+Binv = np.linalg.inv(structure.cell)
+frac_coordinates = structure.arrays['positions'].dot(Binv)
+
+with open(inpFilename, "w+") as f:
+    natoms = len(structure.arrays['numbers'])
+    f.write(structureFormula+"\r\n")
+    f.write("&input film=F /\r\n")
```

Comment:
I don't remember exactly, But I think on MacOS there are issues with the arguments to the namelist not in quotes

Reply:
I can use the `argparse` package.

Reply:
No I meant on the line `&input film=F /`. I remember there were some issues were the inpgen failed when it was not `film="F"`

Reply:
ah, i see. That's an easy fix :)

Reply:
I'm really sorry, but I made a mistake I just tested it and it only works without quotes. I must have remembered the issue wrong. Otherwise this fine to merge

Reply:
No problem, I will revert the changes